### DISCUSSION TITLE
what do I need to do before I run vasp_gibbs?

### DISCUSSION THREAD

**Discussion by likenihood** [ftherrien/VaspGibbs#5] [VaspGibbs Forum]
Should I build supercell for unit cell before run SCF calculation or a relaxation?
 can this code calculate the gibbs free energy for solid matter?

**Comment by ftherrien** [comment]
Hi @likenihood, you don't need to do any preprocessing before running VaspGibbs. You just need to start from a completed Vasp calculation.

It can't be used for solid state systems yet (involving phonons and the Murnaghan equation of state), but I may implement it if there is interest.

**Comment by SubhasmitaRay** [comment]
Hi, I want to install vasp_gibbs in the server. I am getting the error of distribution not found. What can be done in this case? I have a very large system and it is difficult to run in a normal computer. Kindly reply.

**Reply by ftherrien** [reply]
Hi @SubhasmitaRay,  could you give me the exact error message you are getting and what you are doing to get it?

**Comment by SubhasmitaRay** [comment]
Hi Sir,
I installed it properly in the server. I am trying to calculate the Gibbs
free energy and delta G. Can you please tell me what formalism you use to
calculate the Gibbs free energy. I just want to analyse the each parameters
in the Vaspgibbs.md file and accordingly my results. Please share the pdf
or something where you have explained the each parameters and physics
behind the same. Kindly share.
I want to study this properly to carry out my research work.


Regards,
Subhasmita Ray

**Reply by ftherrien** [reply]
Take a look at these [online ressources](https://github.com/ftherrien/VaspGibbs#online-ressources). I followed the same formalism as them. I am working on documentation that will be available here soon.

**Comment by SubhasmitaRay** [comment]
Thanks, I'll check them out.
